### DISCUSSION TITLE
Google Analytics set up for DDaT Playbook

### DISCUSSION THREAD

**Discussion by paigedoutel** [nhsbsa/nhsbsa-digital-playbook#255] [General Playbook Ideas]
I have raised the request to get Google Analytics set up for the DDaT Playbook, as discussed in Playbook working group 16/11/2023. 

The foundation for analytics has now been set up and I need some help from one of the Dev representatives (@yantantether FYI) to add the following tracking code into the head of the website code:

```
<!-- Google tag (gtag.js) -->
<script async src="[https://www.googletagmanager.com/gtag/js?id=G-NQ562NM01Z"></script>](https://www.googletagmanager.com/gtag/js?id=G-NQ562NM01Z%22%3E%3C/script%3E)
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());
 
  gtag('config', 'G-NQ562NM01Z');
</script>
```

Thank you!

**Comment by yantantether** [comment]
Note that when adding this, we will also need our cookie consent library adding

https://gitlab.com/nhsbsa/libraries/node/cookie-component
